blobfuse/blobfuse.cpp

```diff
@@ -12,9 +12,6 @@
 #include <include/DataLakeBfsClient.h>
 #include <include/AttrCacheBfsClient.h>
 
-
-bool gEncodeFullFileName = true;
```

Comment:
This variable is declared as extern in cpplite. we need to remove that as well otherwise compilation will fail at some point.

Reply:
removed, done